build/cloud/03-clusterrole.yaml

```diff
@@ -33,3 +36,6 @@ rules:
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["networking.istio.io"]
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
```

Comment:
Istio related apis is not the general part of the kubeedge, so you can keep it in your own branch  :)

Reply:
> Istio related apis is not the general part of the kubeedge, so you can keep it in your own branch :)

I think this can be filled in edgemesh helm configuration. @Poorunga

Reply:
Maybe we need to patch it when we want to use edgemesh.

Reply:
cc @Poorunga Now edgemesh will use istio api?

Reply:
I added it because I saw related errors in the CloudCore logs.

Reply:
@fisherxu Yes.

And there are many edgemesh users who have encountered this problem, such as: 
https://github.com/kubeedge/edgemesh/issues/93
https://github.com/kubeedge/edgemesh/issues/241